_licenses/postgresql.txt

```diff
@@ -0,0 +1,50 @@
+---
+title: PostgreSQL License
+spdx-id: PostgreSQL
+source: https://opensource.org/licenses/PostgreSQL
+featured: false
+hidden: true
+
+description: A very short, BSD-style license, used specifically for PostgreSQL.  
+
+how: To use it, say that it is The PostgreSQL License, and then substitute the copyright year and name of the copyright holder into the body of the license. Then put the license into a prominent file ("COPYRIGHT", "LICENSE" or "COPYING" are common names for this file) in your software distribution.
+
+using:
+  - PostgreSQL: https://www.postgresql.org/about/licence/
+  - PGAudit: https://github.com/pgaudit/pgaudit/blob/master/LICENSE
+  - .NET Access to PostgreSQL: https://github.com/npgsql/npgsql/blob/dev/LICENSE.txt
+
+permissions:
+  - commercial-use
+  - modifications
+  - distribution
+  - private-use
+
+conditions:
+  - include-copyright
+
+limitations:
+  - liability
+  - warranty
+
+---
+
+The PostgreSQL Licence (PostgreSQL)
+
+Copyright (c) [year] [fullname]
+
+Permission to use, copy, modify, and distribute this software and its
+documentation for any purpose, without fee, and without a written agreement is
+hereby granted, provided that the above copyright notice and this paragraph
+and the following two paragraphs appear in all copies.
+
+IN NO EVENT SHALL [fullname] BE LIABLE TO ANY PARTY FOR DIRECT, INDIRECT,
```

Comment:
Some projects:
1. mention "Postgresql Licen**c**e" (not licen**s**e) and add a simple link to the license, such as  [pgAudit](https://github.com/pgaudit/pgaudit_analyze/blob/master/LICENSE)
1. don't mention "Postgresql" at all and provide the full license with "fullname" substitution, such as [pgBadger](https://github.com/dalibo/pgbadger/blob/master/LICENSE) or [pgAdmin4](https://github.com/postgres/pgadmin4/blob/master/LICENSE)

With Licensee, I tried different license declarations to try to match those situations: only pgBadger was recognized :-/

Notice that removing the `[fullname]` field improve the Licensee's Confidence rate, but only pgBadger is still recognized.

Honestly, I don't know what to do.